File: c/udemy/6-vetores/soma_vetores.c
// Faça um programa para ler dois vetores A e B, contendo N elementos cada. Em seguida, gere um terceiro vetor C onde cada elemento de C é a soma dos elementos correspondentes de A e B. Imprima o vetor C gerado.

#include <stdio.h>

int main() {

    int n;

    printf("Quantos valores? ");
    scanf("%d", &n);

    int a[n], b[n], c[n];

    printf("Valores de A:\n");
    for (int i=0; i<n; i++) {
        scanf("%d", &a[i]);
    }

    printf("Valores de B:\n");
    for (int i=0; i<n; i++) {
        scanf("%d", &b[i]);
    }

    for (int i=0; i<n; i++) {
        c[i] = a[i] + b[i];
    }

    printf("Vetor resultante C:\n");
    for (int i = 0; i < n; i++) {
        printf("%d ", c[i]);
    }
    printf("\n");

    return 0;

}

File: python/udemy/6-vetores/soma_vetores.py
# Faça um programa para ler dois vetores A e B, contendo N elementos cada. Em seguida, gere um terceiro vetor C onde cada elemento de C é a soma dos elementos correspondentes de A e B. Imprima o vetor C gerado.

a = []
b = []
c = []

n = int(input('Quantos valores em cada vetor? '))
for i in range(n):
    num = float(input(f'Valor {i+1} do vetor A: '))
    a.append(num)

for i in range(n):
    num = float(input(f'Valor {i+1} do vetor B: '))
    b.append(num)

for i in range(n):
    ele_c = a[i] + b[i]
    c.append(ele_c)

print(f'Vetor resultante: {c}')